README.md

```diff
@@ -1,13 +1,36 @@
 # Springroll Seed
 
-Springroll Seed is intendend to be used as starting point for games projects. It comes bundled with Webpack, Babel, as well as a few other node modules to streamline development.
+Springroll Seed is intended to be used as starting point for games projects. It comes bundled with Webpack, Babel, as well as a few other node modules to streamline development.
 
 ## Commands
 
-## npm start
+SpringRoll Seed comes with only two commands
+
+### npm start
 
 Starts the dev server
 
-## npm run build
+### npm run build
```

Comment:
Traditionally, SpringRoll has had a concept of "RELEASE" versus "DEBUG" builds. Should we maintain this concept (maybe source maps versus no) and have a task for that: `npm run build-debug`?